lib/pbench/cli/server/user_delete.py

```diff
@@ -0,0 +1,65 @@
+import click
+from pbench import BadConfig
+from pbench.server import PbenchServerConfig
+from pbench.cli.server import pass_cli_context
+from pbench.cli.server.options import common_options
+from pbench.common.logger import get_pbench_logger
+from pbench.server.database.models.users import User
+from pbench.server.database.database import Database
+
+_NAME_ = "pbench-user-delete"
+
+
+class UserDelete:
+    def __init__(self, context):
+        self.context = context
+
+    def execute(self):
+        config = PbenchServerConfig(self.context.config)
+
+        logger = get_pbench_logger(_NAME_, config)
+
+        # We're going to need the Postgres DB to track dataset state, so setup
+        # DB access.
+        Database.init_db(config, logger)
+
+        # Determine if the token belongs to the admin user
+        user = User.query(username=self.context.username)
+
+        # Check if the user has the right privileges
+        if user is None or not user.is_admin():
+            raise RuntimeError(f"Not authorized to delete user {self.context.username}")
+
+        # If the authorization passed delete the user from the given username
+        User.delete(username=self.context.user_to_delete)
+
+        click.echo(f"User {self.context.user_to_delete} deleted")
+
+
+@click.command()
+@common_options
+@click.option(
+    "--username",
+    required=True,
+    prompt=True,
+    help="pbench server account username (will prompt if unspecified)",
+)
+@click.option(
+    "--user-to-delete",
+    prompt=True,
+    required=True,
+    help="Specify username of a pbench server account user to delete (will prompt if unspecified)",
+)
+@pass_cli_context
+def main(context: object, username: str, user_to_delete: str) -> None:
+    # User need to have an admin role to delete another user
+    context.username = username
+    context.user_to_delete = user_to_delete
+
+    try:
+        rv = UserDelete(context).execute()
+    except Exception as exc:
+        click.echo(exc, err=True)
+        rv = 2 if isinstance(exc) == BadConfig else 1
```

Comment:
`isinstance()` should take _two_ arguments (I suspect that the exception that it raises due to the missing argument goes unnoticed, since the deletion has already failed, we've already printed out the appropriate message, and now we are already in the throes of death... :-) ).

Reply:
Ditto the other files.